CHANGELOG.md

```diff
@@ -3,6 +3,10 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## 7.0.1 - TBD
+### Changed
+- Relocated Kafka libraries in `kafka-metastore-listener` under `com.expediagroup.apiary.extensions.shaded.org.apache.kafka` to avoid clashes when moving other components to newer version of Kafka.
```

Comment:
```suggestion
- Relocated Kafka libraries in `kafka-metastore-listener` under `com.expediagroup.apiary.extensions.shaded.org.apache.kafka` to avoid clashes when moving other components to a newer version of Kafka.
```